mmv1/templates/terraform/custom_delete/storage_folder_delete.go.tmpl

```diff
@@ -55,7 +55,14 @@ for deleteObjectError == nil {
 }
 
 err = retry.Retry(1*time.Minute, func() *retry.RetryError {
-    err := config.NewStorageClient(userAgent).Folders.Delete(bucket, name).Do()
+    deleteCall := config.NewStorageClient(userAgent).Folders.Delete(bucket, name)
+    headers := make(http.Header)
+    if v_headers, ok := d.GetOk("custom_headers"); ok {
+    for k, v := range v_headers.(map[string]interface{}) {
+            headers.Add(k, v.(string))
+        }
+    }
+    err = deleteCall.Do()
```

Comment:
Since this function is implemented using the client library the headers object is not used. We will need to pass headers to the client library.